.travis.yml

```diff
@@ -12,18 +12,28 @@ python:
   - 2.7
 
 env:
-  - DJANGO_VERSION=1.4
-  - DJANGO_VERSION=1.5
-  - DJANGO_VERSION=1.6
+  - DJANGO_VERSION=1.4 PYMONGO_VERSION=2.8.1
+  - DJANGO_VERSION=1.4 PYMONGO_VERSION=3.0.2
+  - DJANGO_VERSION=1.5 PYMONGO_VERSION=2.8.1
+  - DJANGO_VERSION=1.5 PYMONGO_VERSION=3.0.2
+  - DJANGO_VERSION=1.6 PYMONGO_VERSION=2.8.1
+  - DJANGO_VERSION=1.6 PYMONGO_VERSION=3.0.2
+  - DJANGO_VERSION=1.7 PYMONGO_VERSION=2.8.1
+  - DJANGO_VERSION=1.7 PYMONGO_VERSION=3.0.2
```

Comment:
Not yet (am planning to look at that next). I've added it to the allowable failures for now